openff/toolkit/utils/toolkit_registry.py

```diff
@@ -403,3 +403,6 @@ def _toolkit_registry_manager(toolkit_registry: Union[ToolkitRegistry, ToolkitWr
             GLOBAL_TOOLKIT_REGISTRY.deregister_toolkit(toolkit)
         for toolkit in original_toolkits:
             GLOBAL_TOOLKIT_REGISTRY.register_toolkit(toolkit)
+
+
+_toolkit_registry_manager = toolkit_registry_manager
```

Comment:
This seems like a simpler implementation than having a public context manager call a private context manager to do (literally) the same thing.

Reply:
Agreed!